extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "PTC_onshape_metadata glTF extension",
+    "type": "object",
+    "description": "glTF extension for specifying IDs and visibility.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "id": {
+            "anyOf":[
+                {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                {
+                    "type": "string",
+                    "minLength": 1
+                }
+            ],
+            "description": "An identifier for this object."
+        },
+        "hidden": {
+            "type": "boolean",
+            "description": "If true, the object should not be rendered."
+        },
```

Comment:
How about: "If true, the object and its descendants should not be rendered."

Reply:
Updated.